src/styles/components/buttons.scss

```diff
@@ -22,14 +21,19 @@
     };
 }
 
+.btn-small {
+    font-weight: 700;
+    padding: 0.5rem 0.75rem;
+}
+
 .btn-large {
     padding: 2.5rem;
 }
 
-@each $name, $type in $os-colors {
+@each $name, $type in $color-types {
     .btn-#{$name} {
         background-color: color($name);
-        color: color(neutral-lightest);
+        color: color($name, 'overlay');
```

Comment:
This line should be reverted.  It looks like it was mistakenly pulled in from one of the merges with one of Roy's old PRs.